### PR TITLE
Npm IT with TestDeployment

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #855 
Remade `NpmITCase` to use `TestDeployment`

### CODE REVIEW

File: src/test/java/com/artipie/npm/NpmITCase.java [hunk 1]
```diff
@@ -60,183 +49,100 @@
...
-            this.cntn.execStdout(
-                "npm", "publish", NpmITCase.PROJ,
-                "--registry", this.url.string(anonymous)
+        this.containers.assertExec(
+            "Package was not installed",
+            new MavenITCase.ContainerResultMatcher(
+                new IsEqual<>(0),
+                new StringContainsInOrder(
+                    Arrays.asList(
+                        String.format("+ %s@1.0.1", NpmITCase.PROJ),
```
Comment (genryxy) [inline src/test/java/com/artipie/npm/NpmITCase.java:93]: @olenagerasimova I'd suggest to extract this string as it is used twice

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one minor suggestion

**Comment by olenagerasimova** [comment]
@genryxy thanks, extracted

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, approve

**Pull request merged**